lib/registry.js

```diff
@@ -258,12 +259,21 @@ Registry.prototype._defineRemoteMethods = function(ModelCtor, methods) {
 
   Object.keys(methods).forEach(function(key) {
     var meta = methods[key];
+    var m = key.match(/^prototype\.(.*)$/);
+    var isStatic = !m;
+
     if (typeof meta.isStatic !== 'boolean') {
-      console.warn('Remoting metadata for "%s.%s" is missing "isStatic" ' +
-        'flag, the method is registered as an instance method.',
-        ModelCtor.modelName,
-        key);
-      console.warn('This behaviour may change in the next major version.');
+      key = isStatic ? key : m[1];
+      meta.isStatic = isStatic;
+    } else {
+      if (meta.isStatic && m) {
```

Comment:
Nitpick: the new block `{...}` starting on L268 is not necessary, it adds extraneous nesting. Here is a version that I consider cleaner:

```js
if (typeof meta.isStatic !== 'boolean') {
  // ...
} else if (meta.isStatic && m) {
  // ...
} else {
  // ...
}
```